sdk/common/src/main/java/com/mesosphere/sdk/testutils/TestConstants.java

```diff
@@ -39,6 +39,11 @@
     public static final Integer PORT_API_VALUE = 8080;
     public static final String HAS_DYNAMIC_PORT_ASSIGNMENT_LABEL = "has-dynamic-port-assignment";
     public static final String HAS_VIP_LABEL = "hasvip";
+    // CNI port mapping constants
+    public static final String EXPECTED_NETWORK_NAME = "dcos";
```

Comment:
Since it's understood that we'll both be creating fake objects with these values and then testing that the protobufs still contain them on the other end, probably no need to prefix them with `EXPECTED_`.